### ISSUE TITLE
GrapesJS background element does not work

### ISSUE THREAD

**Issue by Ameeko** [aimeos/ai-cms-grapesjs#21]
Hello!

The background element does not work.

1) I uploaded an image on the media tab (test.jpg)
2) Save
3) I created a background element on the content tab
4) Save
5) I clicked on the background element and selected test.jpg
6) Save

The generated images have been created, but the background image does not appear on the frontend.

![image](https://user-images.githubusercontent.com/61547701/155205268-86734a42-afeb-4bc6-b3dc-39ea6fc4b815.png)

![image](https://user-images.githubusercontent.com/61547701/155205959-704c1c22-1bd7-488f-bd20-9945ab38cd07.png)

What can cause this error?
It used to work, but I can’t figure out why it doesn’t work now.

**Comment by aimeos** [comment]
If the image was smaller than the viewport width, then no background was assigned to the element. This has been fixed and is available when using:
```
composer req aimeos/ai-client-html:2021.10.x-dev
```

**Comment by Ameeko** [comment]
Hello!
Thanks for the reply!
Unfortunately it doesn't work after the update.

**Comment by aimeos** [comment]
Please make sure you have this changes in you ./public/vendor/shop/themes/default/aimeos.js file:
https://github.com/aimeos/ai-client-html/commit/7337e6f16b438302b4a42f5fef8e996ce3f0e3c7

**Comment by Ameeko** [comment]
Thanks for the reply. It works.
I have my own theme and the aimeos.js in its folder was not up to date.
I use a copy of the default theme and there are changes in aimeos.js, so I always have to merge aimeos-js manually.

**Comment by Ameeko** [comment]
I think it's working wrong.

I uploaded an image (width: 900px)
The program generated 2 images from it: 480w and 900w
I set it as a background.
On the frontend, if I look at the background at 933px width, it still uses the 480w image, which makes the image quality very poor.
![image](https://user-images.githubusercontent.com/61547701/155849288-4594bb07-2962-4110-85e6-ab5f081c4588.png)

![image](https://user-images.githubusercontent.com/61547701/155849491-cf3a17ce-d5b1-4bb5-9a29-27dac79c3a88.png)

**When do we use the small picture and when the big one?**
